add small blurb about git context (#509)
apollographql/docs: src/content/graphos/delivery/schema-checks.mdx

> If you've [integrated schema checks with your GitHub PRs](#integrating-with-github), the "Details" link in your GitHub check takes you to this same details page.

> Additionally, if you run a check within a Git repository, `rover subgraph check` will include both the commit hash and the author of said hash to GraphOS to display on the check.

Reviewer: ```suggestion
Additionally, if you run a check within a Git repository, `rover subgraph check` will include both the commit hash and that hash's author to GraphOS to display on the check.
```
